yarn-project/end-to-end/src/e2e_fees/private_refunds.test.ts

```diff
@@ -153,7 +153,7 @@ describe('e2e_fees/private_refunds', () => {
             aliceRandomness,
             bobRandomness,
             t.bobWallet.getAddress(), // Bob is the recipient of the fee notes.
-            true, // We set max fee/funded amount to zero to trigger the error.
+            true, // We set max fee/funded amount to 1 to trigger the error.
```

Comment:
Had to bump this to 1 instead because otherwise the test would revert [here](https://github.com/AztecProtocol/aztec-packages/blob/5827222af37981ba1eba7dfb0b9afbe766bedc93/noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/main.nr#L383) and not in the expected assert.

Reply:
This is rather yucky, we should prioritze https://github.com/AztecProtocol/aztec-packages/issues/7694